timeseries/src/autogluon/timeseries/models/gluonts/abstract_gluonts.py

```diff
@@ -204,7 +204,7 @@ def load(cls, path: str, reset_paths: bool = True, verbose: bool = True) -> "Abs
             model = load_pkl.load(path=os.path.join(path, cls.model_file_name), verbose=verbose)
             if reset_paths:
                 model.set_contexts(path)
-            model.gts_predictor = PyTorchPredictor.deserialize(Path(path) / cls.gluonts_model_path)
+            model.gts_predictor = PyTorchPredictor.deserialize(Path(path) / cls.gluonts_model_path, device="auto")
```

Comment:
I couldn't figure out a way to test that this works. Modifying `CUDA_VISIBLE_DEVICES` after torch has been imported has no effect, so I couldn't disable CUDA inside the tests. Do you think it's fine if we leave this untested?

Reply:
Assuming you've tested offline, sure.